packages/api/src/event/medical/canvas-event-listener.ts

```diff
@@ -0,0 +1,51 @@
+import CanvasSDK from "@metriport/core/external/canvas/index";
+import { createFullNote } from "@metriport/core/external/canvas/note";
+import { getEnvVarOrFail } from "@metriport/shared";
+import { PatientEvents, patientEvents, PatientEvent } from "./patient-event";
+import { getPatientOrFail } from "../../command/medical/patient/get-patient";
+import { Util } from "../../shared/util";
+
+const log = Util.log(`[CANVAS EVENT LISTENER]`);
```

Comment:
This is deprecated, let's use `import { out } from "@metriport/core/util/log";` instead please

Also, nit: no need for brackets